### PR TITLE
Admin: add suppliers to product list by default

### PR DESCRIPTION
**Pull request by tulimaki**
When multiple suppliers are enabled add suppliers column to default column. Also ensure that when suppliers is used as column the filtering by name is also set for the column.

### CODE REVIEW

File: shuup/admin/modules/products/views/list.py [hunk 1]
```diff
@@ -103,9 +105,34 @@ class ProductListView(PicotableListView):
...
+            self.default_columns.append(
+                Column(
+                    "suppliers",
+                    _("Suppliers"),
+                    display="format_suppliers",
+                    ordering=8,
+                    filter_config=get_suppliers_filter(),
+                )
+            )
+        super(ProductListView, self).__init__()
```
Comment (chessbr) [inline shuup/admin/modules/products/views/list.py]: are we officially dropping py 27?

Comment (tulimaki) [reply shuup/admin/modules/products/views/list.py]: Not just yet.

Comment (chessbr) [reply shuup/admin/modules/products/views/list.py]: so this needs fixing

Comment (tulimaki) [reply shuup/admin/modules/products/views/list.py]: how is this not py27 compatible?

Comment (chessbr) [reply shuup/admin/modules/products/views/list.py]: :man_facepalming: that is right, Monday...

**Pull request merged**